src/redux/actionConstants.js

```diff
@@ -0,0 +1,5 @@
+export const SET_DATA = 'data/set_data'
```

Comment:
Would move this file to actions folder

Reply:
not sure if it's necessary, but - done ✅